torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4792,6 +4792,12 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            sample_inputs_func=sample_inputs_pow,
            supports_inplace_autograd=False,
            assert_autodiffed=True),
+    OpInfo('float_power',
+           dtypes=all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool),
+           backward_dtypes=all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool),
+           sample_inputs_func=sample_inputs_pow,
```

Comment:
Since `float_power` essentially calls `pow` with tensors casted to double data types (double/complex double), so the sample inputs can be shared between `pow` and `float_power`. This is from my understanding, please correct if I'm wrong.

Reply:
Note to self: Check this later.